Scripts/DCS-BIOS/lib/modules/aircraft_modules/FW-190D9.lua

```diff
@@ -0,0 +1,224 @@
+FW_190A8:defineRotary("ALT_PRESS_SET", 10, 3001, 34, "Cockpit", "Altimeter Pressure Set")
+FW_190A8:defineToggleSwitch("FLAPS_UP_TOGGLE", 8, 3001, 62, "Cockpit", "Flaps Up")
+FW_190A8:defineToggleSwitch("FLAPS_TO", 8, 3002, 63, "Cockpit", "Flaps Take Off")
+FW_190A8:defineToggleSwitch("FLAPS_LAND", 8, 3003, 64, "Cockpit", "Flaps Landing")
+FW_190A8:defineToggleSwitch("LG_RET", 8, 3005, 65, "Cockpit", "Landing Gears Retraction")
+FW_190A8:defineToggleSwitch("LG_RET_CVR", 8, 3006, 66, "Cockpit", "Landing Gears Retraction Cover")
+FW_190A8:defineToggleSwitch("LG_EXT", 8, 3007, 67, "Cockpit", "Landing Gears Extending")
+FW_190A8:defineRotary("TURN_BEZ", 13, 3001, 24, "Cockpit", "Turnable Bezel")
+FW_190A8:defineRotary("WND_SET", 13, 3004, 25, "Cockpit", "Wind/Set Knob")
+FW_190A8:definePushButton("START_STOP", 13, 3006, 26, "Cockpit", "Start - Stop Button")
+FW_190A8:definePushButton("STOPWATCH", 13, 3008, 28, "Cockpit", "Stopwatch Button")
+FW_190A8:defineToggleSwitch("MASTER_ARM", 20, 3003, 48, "Weapon Control", "Master Arm")
+FW_190A8:defineRotary("GUN1_AMM_C", 20, 3004, 51, "Weapon Control", "Set Gun 1 Ammunition Count")
+FW_190A8:defineRotary("GUN2_AMM_C", 20, 3005, 54, "Weapon Control", "Set Gun 2 Ammunition Count")
+FW_190A8:defineRotary("GUN3_AMM_C", 20, 3006, 57, "Weapon Control", "Set Gun 3 Ammunition Count")
+FW_190A8:defineRotary("GUN4_AMM_C", 20, 3007, 60, "Weapon Control", "Set Gun 4 Ammunition Count")
+FW_190A8:defineToggleSwitch("MAIN_RKT", 20, 3008, 107, "Weapon Control", "Main Rocket Switch")
+FW_190A8:defineToggleSwitch("RKT_EMG_REL_CVR", 20, 3009, 137, "Weapon Control", "Rocket Emergency Release Cover")
+FW_190A8:defineToggleSwitch("RKT_EMG_REL", 20, 3010, 108, "Weapon Control", "Rocket Emergency Release")
+FW_190A8:definePushButton("JETT_FUS_STORES", 20, 3012, 92, "Cockpit", "Jettison Fuselage Stores")
+FW_190A8:defineToggleSwitch("SIGHT_GYRO", 21, 3001, 132, "Gun Sight", "EZ42 Gunsight Gyro Power Switch, ON/OFF")
+FW_190A8:definePotentiometer("SIGHT_WING_SPAN", 21, 3002, 131, { 0, 1 }, "Gun Sight", "EZ42 Gunsight Target Wingspan Knob")
+FW_190A8:defineToggleSwitch("SIGHT_NIGHT", 21, 3011, 134, "Gun Sight", "EZ42 Gunsight Night Lens Filter, ON/OFF")
+FW_190A8:defineRotary("SIGHT_DISC_CTRL", 21, 3003, 4, "Gun Sight", "EZ42 Gunsight Target Distance Control")
+FW_190A8:defineRotary("SIGHT_BRIGHT", 21, 3004, 135, "Gun Sight", "EZ42 Gunsight Brightness Control")
+FW_190A8:defineRotary("SIGHT_ALT_KNOB", 21, 3012, 191, "Gun Sight", "EZ42 Gunsight Altitude Aiming Correction Adjusting Knob I")
+FW_190A8:defineRotary("INSTR_LGHT_CTRL", 7, 3001, 78, "Cockpit", "Instrument Lights Brightness")
+FW_190A8:defineRotary("FUG16_VOLUME", 15, 3003, 83, "Radio", "Radio Volume")
+FW_190A8:defineToggleSwitch("FT_ZF_SWITCH", 15, 3006, 84, "Radio", "FT FT / Y ZF Mode Switch")
+-- Allocate 16 bits of address space to avoid changing the address of subsequent allocations:
+FW_190A8:allocateInt(65535)
```

Comment:
This function shouldn't be public I don't think. Module:reserveIntValue is public and does the same thing - it's just a bit clearer what it's doing.